app/src/Components/NavBar/Nav.tsx

```diff
@@ -57,8 +57,10 @@ function TopNav ({ onSearchChange, onSortClick, onOrderClick, onMinChange, onMax
     onMaxChange(maxPrice);
   }
 
-  // Disable the filter submit button when minPrice > maxPrice and there is a value for each of the min and max prices
-  let isFilterSubmitDisabled: boolean = Number(minPrice) > Number(maxPrice) && minPrice !== "" && maxPrice !== "";
+  // Disable the filter submit button when minPrice is not less than maxPrice and there is a value for each of the min and max prices
+  // Also disable is min or max is a negative value.
+  let isFilterSubmitDisabled: boolean = (!(Number(minPrice) < Number(maxPrice)) && minPrice !== "" && maxPrice !== "")
```

Comment:
Small change, but I think it'd also be nice to have it so that the user can still search for cards if the minPrice <= maxPrice. That way, user can input minPrice = $10 and maxPrice = $10, and they would get all the cards that are $10 exactly

Reply:
Yeah that is good I did not think of that.